docs/tutorials/metrics_hammingloss.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
<p>I do not see this button in our template. Is it added on your own or where do you can this?</p><p> </p><p><a href="https://github.com/tensorflow/addons/blob/master/docs/tutorials/_template.ipynb" target="_blank">https://github.com/tensorflow/addons/blob/master/docs/tutorials/_template.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/2044/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
@harshithere here it is.